sbc_compassion/tests/test_tools.py

```diff
@@ -121,7 +121,8 @@ def _read_img(self, path):
 
     def _pattern_recognition(self, path):
         img = self._read_img(path)
-        return tools.patternrecognition.find_template(img, self.templates, 0.3)
+        return tools.patternrecognition.find_template(
+            img, self.templates, 0.3)[:2]
```

Comment:
If its always None then you need to update `find_template` method and remove that return.